db/seeds.rb

```diff
@@ -10,8 +10,8 @@
   )
 
   # Confirm admin account
-  admin.update(confirmed_at: Time.now)
-  admin.level_set(:mail)
+  admin.update(confirmed_at: Time.now, state: 'active')
```

Comment:
you can use `admin.confirm` method instead of `admin.update(confirmed_at: Time.now, state: 'active')`